test/ducks/apps/components/applicationPage/__snapshots__/index.spec.js.snap

```diff
@@ -21,7 +21,7 @@ exports[`ApplicationPage component should be rendered correctly with app infos f
     >
       <DefaultButton
         className="sto-app-back"
-        icon="left"
+        icon={[Function]}
```

Comment:
I recommend mocking the Left for this snap and verify Left mock is present in the snap